.github/workflows/log-viewer-docs.yaml

```diff
@@ -2,6 +2,8 @@ name: "docs"
 
 on:
   pull_request:
+    paths:
```

Comment:
shall we include other docs related source files:
1. `tools/**/*`
2. `.gitmodules`
3. `taskfile.yaml`
4. i haven't looked up any GH actions docs, but if we don't include `.github/workflows/log-viewer-docs.yaml`, would a workflow be trigged if we modify the yaml file in a PR?